cmd/minishift/cmd/start.go

```diff
@@ -360,11 +358,12 @@ func calculateDiskSizeInMB(humanReadableDiskSize string) int {
 
 func determineIsoUrl(iso string) string {
 	iso = strings.ToLower(iso)
+	defaultISO := ""
```

Comment:
not sure what you gain here with this variable. I think at least the name of the variable is confusing. It is not _isoNotSpecified_